Website/src/app/components/layout_base.tsx

```diff
@@ -0,0 +1,17 @@
+import '../../components/globals.css'
+import Header from "@/app/components/header";
+import Footer from "@/app/components/footer";
+
+export default function Layout({
```

Comment:
A quick look through this shows me that there are practically no comments in the code and no documentation for the public API. That makes it a lot harder to understand and also review code such as this. Please comment your code.